### ISSUE TITLE
Github Action Deprecation Warnings

### ISSUE THREAD

**Issue by ChristopherChudzicki** [mitodl/open-discussions#4016]
We are getting several github actions deprecation warnings: https://github.com/mitodl/open-discussions/actions/runs/5134607216

We should resolve them.

**Comment by pdpinch** [comment]
Can renovate help with this?

**Comment by ChristopherChudzicki** [comment]
@pdpinch Good point. Having read a bit more about this, I think the answer is "Sometimes". It looks like renovate can bump action versions
```diff
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
```

Renovate
- won't bump things like the NodeJS or Python version ... *There are probably better ways to handle that anyway; E.g., in a github workflow, read the version to use from some settings file*
- as ever, won't fix breaking changes for us ... One of the deprecation warnings is
    > [javascript-tests](https://github.com/mitodl/open-discussions/actions/runs/5134607216/jobs/9238850455#step:5:5)
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
https://docs.renovatebot.com/modules/manager/github-actions

**Comment by JenniWhitman** [comment]
Chris completed this in https://github.com/mitodl/open-discussions/pull/4022
